site/content/3.12/aql/graphs/traversals.md

```diff
@@ -255,8 +110,208 @@ FOR vertex[, edge[, path]]
+- `"global"` – it is guaranteed that each vertex is visited at most once during
+  the traversal, no matter how many paths lead from the start vertex to this one.
+  If you start with a `min depth > 1` a vertex that was found before *min* depth
+  might not be returned at all (it still might be part of a path).
+  It is required to set `order: "bfs"` or `order: "weighted"` because with
+  depth-first search the results would be unpredictable. **Note:**
+  Using this configuration the result is not deterministic any more. If there
+  are multiple paths from *startVertex* to *vertex*, one of those is picked.
+  In case of a `weighted` traversal, the path with the lowest weight is
+  picked, but in case of equal weights it is undefined which one is chosen.
+- `"none"` (default) – no uniqueness check is applied on vertices
+
+#### `uniqueEdges`
+
+Ensure edge uniqueness (string):
+
+- `"path"` (default) – it is guaranteed that there is no path returned with a
+  duplicate edge
+- `"none"` – no uniqueness check is applied on edges. **Note:**
+  Using this configuration, the traversal follows edges in cycles.
+
+#### `edgeCollections`
+
+Restrict edge collections the traversal may visit (string\|array).
+
+If omitted or an empty array is specified, then there are no restrictions.
+
+- A string parameter is treated as the equivalent of an array with a single
+  element.
+- Each element of the array should be a string containing the name of an
+  edge collection.
+
+#### `vertexCollections`
+
+Restrict vertex collections the traversal may visit (string\|array).
+
+If omitted or an empty array is specified, then there are no restrictions.
+
+- A string parameter is treated as the equivalent of an array with a single
+  element.
+- Each element of the array should be a string containing the name of a
+  vertex collection.
+- The starting vertex is always allowed, even if it does not belong to one
+  of the collections specified by a restriction.
+
+#### `parallelism`
+
+{{< tag "ArangoDB Enterprise Edition" "ArangoGraph" >}}
+
+Parallelize traversal execution (number).
+    
+If omitted or set to a value of `1`, the traversal execution is not parallelized.
+If set to a value greater than `1`, then up to that many worker threads can be
+used for concurrently executing the traversal. The value is capped by the number
+of available cores on the target machine.
+
+Parallelizing a traversal is normally useful when there are many inputs (start
+vertices) that the nested traversal can work on concurrently. This is often the
+case when a nested traversal is fed with several tens of thousands of start
+vertices, which can then be distributed randomly to worker threads for parallel
+execution.
+
+#### `maxProjections`
+
+{{< tag "ArangoDB Enterprise Edition" "ArangoGraph" >}}
+
+Specifies the number of document attributes per `FOR` loop to be used as
+projections (number). The default value is `5`.
+
+#### `indexHint` (object, *optional*, i):
```

Comment:
What does the `i` at the end mean here?

Reply:
Dunno, but looks like a typo. Removed the entire part in parens from the headline since all options are optional and the type (object) is mentioned in the text anyway.